cypress/helpers/vault.ts

```diff
@@ -0,0 +1,359 @@
+    name: 'system:auth-delegator',
+  },
+  subjects: [
+    {
+      kind: 'ServiceAccount',
+      name: 'vault',
+      namespace: 'hashicorp',
+    },
+  ],
+};
+
+export const serviceAccountJSON = {
+  apiVersion: 'v1',
+  kind: 'ServiceAccount',
+  metadata: {
+    labels: {
+      'app.kubernetes.io/instance': 'vault',
+      'app.kubernetes.io/name': 'vault',
+    },
+    name: 'vault',
+  },
+};
+
+export const getPVCJSON = {
+  kind: 'PersistentVolumeClaim',
+  apiVersion: 'v1',
+  metadata: {
+    name: 'vault-storage',
+    labels: {
+      'app.kubernetes.io/instance': 'vault',
+      'app.kubernetes.io/name': 'vault',
+    },
+  },
+  spec: {
+    accessModes: ['ReadWriteOnce'],
+    resources: {
+      requests: {
+        storage: '10Gi',
+      },
+    },
+  },
+};
+
+export const configMapJSON = {
+  kind: 'ConfigMap',
+  apiVersion: 'v1',
+  metadata: {
+    name: 'vault-config',
+    labels: {
+      'app.kubernetes.io/instance': 'vault',
+      'app.kubernetes.io/name': 'vault',
+    },
+  },
+  data: {
+    'vault-config':
+      '{"backend": {"file": {"path": "/vault/data"}},' +
+      '"default_lease_ttl": "168h","max_lease_ttl": "720h","disable_mlock": true,' +
+      '"ui": true,"listener": {"tcp": {"address": "0.0.0.0:8200","tls_disable" : true}}}',
+  },
+};
+
+export const serviceJSON = {
+  apiVersion: 'v1',
+  kind: 'Service',
+  metadata: {
+    name: 'vault',
+    annotations: {
+      'service.alpha.openshift.io/serving-cert-secret-name': 'vault-cert',
+    },
+    labels: {
+      'app.kubernetes.io/instance': 'vault',
+      'app.kubernetes.io/name': 'vault',
+    },
+  },
+  spec: {
+    ports: [
+      {
+        name: 'vault',
+        port: 8200,
+      },
+    ],
+    selector: {
+      'app.kubernetes.io/instance': 'vault',
+      'app.kubernetes.io/name': 'vault',
+    },
+    publishNotReadyAddresses: true,
+  },
+};
+
+export const routeJSON = {
+  kind: 'Route',
+  apiVersion: 'route.openshift.io/v1',
+  metadata: {
+    name: 'vault',
+    labels: {
+      'app.kubernetes.io/instance': 'vault',
+      'app.kubernetes.io/name': 'vault',
+    },
+  },
+  spec: {
+    to: {
+      kind: 'Service',
+      name: 'vault',
+      weight: 100,
+    },
+    port: {
+      targetPort: 8200,
+    },
+  },
+};
+
+export const networkPolicyJSON = {
+  apiVersion: 'networking.k8s.io/v1',
+  kind: 'NetworkPolicy',
+  metadata: {
+    name: 'vault',
+    labels: {
+      'app.kubernetes.io/name': 'vault',
+      'app.kubernetes.io/instance': 'vault',
+    },
+  },
+  spec: {
+    podSelector: {
+      matchLabels: {
+        'app.kubernetes.io/instance': 'vault',
+        'app.kubernetes.io/name': 'vault',
+      },
+    },
+    ingress: [
+      {
+        from: [
+          {
+            namespaceSelector: {},
+          },
+        ],
+        ports: [
+          {
+            port: 8200,
+            protocol: 'TCP',
+          },
+        ],
+      },
+    ],
+  },
+};
+
+export const deploymentJson = {
```

Comment:
```suggestion
export const deploymentJSON = {
```
Lets keep the case same. Do we really need to mention these are `JSONs` in the variable name though?